server/service/scenarios/index.js

```diff
@@ -11,12 +11,16 @@ const {
     deleteScenario
 } = require('./endpoints.js');
 
-scenariosRouter.get('/:scenario_id', [getScenario]);
+scenariosRouter.get('/:scenario_id', [lookupScenario(), getScenario]);
 
 scenariosRouter.put('/', [validateRequestBody, addScenario]);
 
-scenariosRouter.post('/:scenario_id', [validateRequestBody, setScenario]);
+scenariosRouter.post('/:scenario_id', [
+    lookupScenario(),
+    validateRequestBody,
+    setScenario
+]);
 
-scenariosRouter.delete('/:scenario_id', [validateRequestBody, deleteScenario]);
```

Comment:
Why don't we need to validate the request body? I guess because we don't have a request body for this route?

Reply:
Because there isn't a body for the DELETE request yeah